packages/indexer/src/indexers/autocrat/autocrat-v1_0-indexer.ts

```diff
@@ -25,34 +29,146 @@ export const AutocratV0_1Indexer: InstructionIndexer<AutocratV1> = {
   },
 };
 
-export const AutocratV0_1AccountIndexer: AccountInfoIndexer = {
-  index: async (
-    accountInfo: AccountInfo<Buffer>,
-    account: PublicKey,
-    context?: Context
-  ) => {
+export const AutocratV1_0DaoIndexer: DaoIndexer = {
```

Comment:
@LukasDeco just wanted to see what you thought about this before I started with the rest. There's certainly issues with relations, but just high level if this makes sense.

Reply:
this looks pretty great honestly. The only thing I would really suggest is to use the `IntervalFetchIndexer` type if this is meant to run this request every X seconds and then we can just pipe that into our already built out workflows. If you have these custom types for each indexer like `DaoIndexer` it will make more custom work for each one which is probably unncessary. Thoughts?